Pyjion/absint.cpp

```diff
@@ -132,6 +132,15 @@ bool AbstractInterpreter::preprocess() {
                     }
                 }
                 break;
+            case LOAD_GLOBAL:
+            {
+                auto name = PyUnicode_AsUTF8(PyTuple_GetItem(m_code->co_names, oparg));
+                if (!strcmp(name, "vars") || !strcmp(name, "dir")) {
```

Comment:
Probably should add a comment that we are aware this fails if either `vars()` or `dirs()` is bound locally, but that's simply a limitation people will have to live with and avoid.